types/module/module.go

```diff
@@ -476,10 +476,6 @@ func (m *Manager) InitGenesis(ctx sdk.Context, cdc codec.JSONCodec, genesisData
 		}
 	}
 
-	// a chain must initialize with a non-empty validator set
```

Comment:
Reposting my comment from here: https://github.com/cosmos/cosmos-sdk/pull/15917#discussion_r1174896201

> I am not that familiar with ICS codebase, but this makes sense.
Have you investigated if the consumer app could recover the panic? I know this isn't ideal, but this is a [UX improvement](https://github.com/cosmos/cosmos-sdk/issues/8961) for all but consumer chains.

In main, it will be skipping this error, but the question still stands.